test_app/resource_api.py

```diff
@@ -0,0 +1,20 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Group
+
+from ansible_base.authentication.models import Authenticator
+from ansible_base.resource_registry.registry import ResourceConfig, ServiceAPIConfig, SharedResource
+from ansible_base.resource_registry.shared_types import TeamType, UserType
+
+
+class APIConfig(ServiceAPIConfig):
+    service_type = "aap"
+
+
+RESOURCE_LIST = (
+    ResourceConfig(get_user_model(), shared_resource=SharedResource(serializer=UserType, is_provider=False), name_field="username"),
+    ResourceConfig(
+        Group,
```

Comment:
the test_app does have a team and organization model. I tested this and I can see a resources entry created when creating a Group, but seeing them for organization & team would be more instructive.

Reply:
oooh, good to know. I can update the tests to use those.